privly.safariextension/privly.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <title>Privly Safari Extension</title>
+  <script type="text/javascript" charset="utf-8">
+    function perform_command(event) {
```

Comment:
I have added another file now scripts/handle_command_event.js which handles the different command events.